<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LadderDiagram</class>
 <widget class="QWidget" name="LadderDiagram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 14px;
	/*background:transparent;*/
}
QWidget#widget{
	border:1px solid white;
	background:transparent;
}

QTableWidget{
	border:none;
	selection-background-color:transparent;
	background:transparent;
}

QTableWidget::item{
	border:none;
}

QTableWidget::item:!active{
	border:none;
}

QTableWidget#tableWidget_input::item:selected{
	border:1px solid yellow;
	border-bottom:none;
}

QTableWidget#tableWidget_output::item:selected{
	border:1px solid yellow;
}

QTableWidget::item:next-selected{
	border-left:none;
}

QTableWidget::item:previous-selected{
	border-right:none;
}

QPushButton:focus{
	padding:-4px;
}

QPushButton#pbt_firstPage:focus,QPushButton#pbt_firstPage:pressed,
QPushButton#pbt_trailerPage:focus,QPushButton#pbt_trailerPage:pressed,
QPushButton#pbt_prePage:focus,QPushButton#pbt_prePage:pressed,
QPushButton#pbt_nextPage:focus,QPushButton#pbt_nextPage:pressed,
QPushButton#pbt_clearSearch:focus,QPushButton#pbt_clearSearch:pressed,
QPushButton#pbt_searchInput:focus,QPushButton#pbt_searchInput:pressed,
QPushButton#pbt_searchOutput:focus,QPushButton#pbt_searchOutput:pressed
{
	background:#0000fe;
}

QPushButton#pbt_firstPage, #pbt_trailerPage, #pbt_prePage, #pbt_nextPage,#pbt_clearSearch,#pbt_searchInput,#pbt_searchOutput{
	color:yellow;
	border:1px solid yellow;
}

QLabel#lbl_curPage, #lbl_totalPage,#lbl_searchId{
	color:white;
	border:1px solid white;
}

QComboBox#comboBox{
	background:transparent;
	color:white;
	border:1px solid #fafa00;
	/*border-radius:4px;*/
}
QComboBox#comboBox::down-arrow{
	image: url(:/ComboBox/Image/ComboBox/down_arrow.png);
}

QComboBox#comboBox::drop-down{
	border:none;
}

QComboBox QAbstractItemView {
	background-color: #28231f;
	color: white;
	border:none;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>19</width>
       <height>518</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget"/>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lbl_curPage">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>第1页</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_totalPage">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>共1页</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbt_firstPage">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>首页</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbt_trailerPage">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>尾页</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbt_prePage">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>上一页</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbt_nextPage">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>下一页</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>26</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>S</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>T</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>K</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_searchId">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbt_searchInput">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>查找输入</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbt_searchOutput">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>查找输出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbt_clearSearch">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>19</width>
       <height>518</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
